build.py

```diff
@@ -497,7 +500,7 @@ def _prefix_path(*parts: str) -> str:
             env.update(before)
 
 
-def _likely_binary(sdist: str) -> str | None:
+def _likely_binary(package: Package, sdist: str) -> str | None:
```

Comment:
it'll be much easier to test if this takes `likely_binary_ignore` directly rather than the whole `Package`